pkg/security/secl/model/model.go

```diff
@@ -826,8 +826,9 @@ type DNSEvent struct {
 type BindEvent struct {
 	SyscallEvent
 
-	AddrFamily uint16        `field:"addr.family" constants:"Network Address Family constants"` // Address family
-	Addr       IPPortContext `field:"addr"`                                                     // Bound address
+	Addr       IPPortContext `field:"addr"`          // Bound address
+	AddrFamily uint16        `field:"addr.family"`   // Address family
+	Protocol   uint8         `field:"addr.protocol"` // Socket protocol
```

Comment:
Let's start with not exporting this field

Reply:
I've split the PR in two, the addition of the protocol to bind is now separated.
Thanks